src/backup.sh

```diff
@@ -34,6 +34,14 @@ rm "$local_file"
 
 echo "Backup complete."
 
+if [ -n "$DISCORD_WEBHOOK_URL" ]; then
+  echo "Sending notification to Discord..."
+
+  python3 -c "import requests; requests.post('$DISCORD_WEBHOOK_URL', json={'content': '[**backup**] of $POSTGRES_DATABASE database completed on $timestamp. URL: $s3_uri'})"
```

Comment:
Out of curiosity - doesn't the base image need python3 to be installed? I am not sure if Alpine comes with it out of the box. Maybe it does. This just made me smile because README mentions this image has no dependency on python.

Reply:
[Python3 is already installed on the image.](https://github.com/eeshugerman/postgres-backup-s3/blob/16df9b3980cb2af232e555972e86a7e23f9fb3b7/src/install.sh#L14-L16) It's a dependency of the AWS CLI. The README states there is no dependency Python **2**.